astropy/nddata/utils.py

```diff
@@ -90,11 +90,11 @@ def overlap_slices(large_array_shape, small_array_shape, position,
 
     Returns
     -------
-    slices_large : tuple of slices
+    slices_large : tuple of slice
```

Comment:
Does "slice" (without backticks) get turned into the builtin `slice()` function? If not, keep as "slices"?

Reply:
yeah, it links to the builtin.

Reply:
... double checking

Reply:
👍. links to builtin